streamline/README.md

```diff
@@ -1,215 +1,97 @@
 # ACM for Nitro Enclaves Streamline
 
-A CDK app that automates the installation and configuration of ACM for Nitro Enclaves on Amazon EC2 instances. This project simplifies the complex [manual installation steps](https://docs.aws.amazon.com/enclaves/latest/user/install-acm.html) by breaking them down into three modular stacks.
+A CDK app that automates the installation and configuration of ACM for Nitro Enclaves on Amazon EC2 instances. This project simplifies the complex [manual installation process](https://docs.aws.amazon.com/enclaves/latest/user/install-acm.html) by providing both a simple CLI tool (`acmne-cli`) and three modular CDK stacks. 
 
 ## Architecture Overview
 The app consists of **three** main CDK stacks:
 
 ![nitro_enclaves_acm_streamline](assets/images/nitro_enclaves_acm_streamline.svg)
 
 ### 1. Certificate Stack (Step 1)
```

Comment:
We probably want some comment above that the `Step x` here and below refer to the steps from the manual documentation. Could be confusing without that.